examples/3d/ssao.rs

```diff
@@ -39,8 +36,8 @@ fn setup(
             msaa: Msaa::Off,
             ..default()
         })
-        .insert(ScreenSpaceAmbientOcclusionBundle::default())
-        .insert(TemporalAntiAliasBundle::default());
+        .insert(ScreenSpaceAmbientOcclusion::default())
```

Comment:
Merging these with the `spawn` also would've been nice. tho that might conflict with a PR for `Camera3dBundle` if anyone is working on one

Reply:
Yeah, merged these now. I do have a camera migration basically ready, but I'm not sure if the `Camera` vs. `CameraSettings` vs. something else was fully decided yet.